.github/workflows/ci.yaml

```diff
@@ -3,26 +3,12 @@ on:
   pull_request:
   workflow_dispatch:
 jobs:
-  # see https://github.com/marketplace/actions/semantic-pull-request
-  lint-PR:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: amannn/action-semantic-pull-request@v5.3.0
```

Comment:
Think we'll need to remove this as a required check before merging:

![image](https://github.com/guardian/cdk/assets/836140/cb6ca2b8-b85e-4759-8ffa-1604a82f2499)

Reply:
Yeah I noticed that - I can't change settings on this repo, are you ok to do that please?

Reply:
Done.

Reply:
Thanks!